sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/IssueLocationCollector.cs

```diff
@@ -60,6 +64,18 @@ public IList<IIssueLocation> GetExpectedIssueLocations(IEnumerable<TextLine> lin
             return MergeLocations(locations, preciseLocations);
         }
 
+        internal IList<IIssueLocation> GetExpectedBuildErrors(IEnumerable<TextLine> lines)
+        {
+            return lines
+                .SelectMany(GetBuildErrorsLocations)
+                .WhereNotNull()
+                .ToList()
```

Comment:
Redundant _.ToList()_ in the middle of the expression (and some redundant extra lines after the method).